packages/appstore/package.json

```diff
@@ -34,6 +34,7 @@
         "@deriv/trader": "^3.8.0",
         "@deriv/translations": "^1.0.0",
         "@deriv/hooks": "^1.0.0",
+        "@deriv/wallets": "^1.0.0",
```

Comment:
Add `@deriv/wallets` as a dependency to `@deriv/appstore`, Later it will be moved to `@deriv/core` instead.